### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by DavidBronfen**
#7

### CODE REVIEW

**Comment by DavidBronfen** [comment]
Trying to see the `<p> header works!</p>` in the app, the HeaderComponent has been included in the `app.module` file.

File: client/src/app/app.module.ts [hunk 1]
```diff
@@ -7,10 +7,12 @@ import { HttpModule } from '@angular/http';
 import { MaterialModule } from '@angular/material';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './src/app/components/header/header.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    HeaderComponent
```
Comment (nirgn975) [inline client/src/app/app.module.ts]: Add a trailing comma.

**Comment by nirgn975** [comment]
> Trying to see the <p> header works!</p> in the app, the HeaderComponent has been included in the app.module file.

@DavidBronfen You should also use this component in some other component. like write 
`<app-header></app-header>` in `app.component.html` file.

**Comment by DavidBronfen** [comment]
Anonymous user icon looks pixelated 
The icon:
anonymous_chief
A screenshot of the header:
image

**Comment by DavidBronfen** [comment]
Removing the default test form the `app.po.ts` and `app.e2e-spec.ts`.

**Comment by DavidBronfen** [comment]
@nirgn975 any suggestion?

**Comment by nirgn975** [comment]
@DavidBronfen Done.
Please look at [the code I changed](https://github.com/DavidBronfen/Retrogusto/pull/9/commits/9a8c4cfce6cf74beb3db87562b6abf1af82c39be) and try to understand why this was the problem.

**Comment by DavidBronfen** [comment]
@nirgn975 I think I understand all the changes you've been made, 
Deleted the tests and added the `NO_ERRORS_SCHEMA` (that i don't realy know what is it but i guess it's dealing with the errors of the test)

**Comment by DavidBronfen** [comment]
@nirgn975 Do you think it would be better to add all the data files (images, icons, etc.) in his own Pull request instead of loading it here?

**Comment by nirgn975** [comment]
When testing a component that has sub-components you have a few options:

1. Import the module that sets up the component and sub-components. Problem is you instantiate the real sub-components and their dependencies.
2. Mock sub-components. This can be tedious depending on how many sub-components it has.
3. Use `NO_ERRORS_SCHEMA` and ignore sub-components.

Given that most of the time you only care about the current component, we use here `NO_ERRORS_SCHEMA`. We care only about the current component because we test the other components separately.

**Comment by DavidBronfen** [comment]
Waiting for #15 to be merged

**Comment by nirgn975** [comment]
@DavidBronfen Is this ready to review?

**Comment by DavidBronfen** [comment]
@nirgn975 Will be tonight.

**Comment by DavidBronfen** [comment]
@nirgn975 This PR should be ready for review.
I've just changed the logo image to `svg`.

**Comment by nirgn975** [comment]
@DavidBronfen But the tests are still failing \:
Actually I'll have more time today, so I will fix those and let's merge this PR already - it's been far too long in the works..

**Pull request merged**